integration/use-cases/create-private-registry.js

```diff
@@ -2,15 +2,28 @@ const axios = require("axios");
 const utils = require("./lib/utils");
 
 test("Creates a private registry", async () => {
+  // ODIC login
+  var token;
+  page.on('response', response => {
+    token = response.headers()["authorization"] || token;
+  });
   await page.goto(getUrl("/#/c/default/ns/default/config/repos"));
+  await page.waitForNavigation();
```

Comment:
I don't atm the environment set up, but as far as I recall, it was getting errors like _Execution context was destroyed, most likely because of a navigation_. This additional wait would solve them.
However, I cannot determine which are strictly necessary and which are not. Next week I can check it if you want.